cloud/app.py

```diff
@@ -42,6 +44,27 @@ def collectBridgeMetrics(devices):
     return bridgedict
 
 
+def token_required(f):
+    @wraps(f)
+    def decorator(*args, **kwargs):
+
+        token = None
+        if 'x-access-tokens' in request.headers:
+            token = request.headers['x-access-tokens']
+
+        if not token:
+            return {'message': 'a valid token is missing'}
+
+        try:
+            data = jwt.decode(token, app.config['SECRET_KEY'], algorithms=["HS256"])
+            if not data['public_id'] in app.config['BRIDGE_TOKENS']:
```

Comment:
Another comment :sweat_smile: Why is your whitelist called `tokens`? It clearly contains IDs.

Reply:
I was thinking at first doing a dictionary with an id and the token

Reply:
Is that still the plan? Isn't it enough that you get the ID when decode the token?

Reply:
yes I can rename it and can only store the id ^^